tests/test_tensor.py

```diff
@@ -252,25 +252,65 @@ def test_pdf_calculations(backend):
     assert tb.tolist(tb.normal_cdf(tb.astensor([0.8]))) == pytest.approx(
         [0.7881446014166034], 1e-07
     )
-    assert tb.tolist(
-        tb.normal_logpdf(
-            tb.astensor([0, 0, 1, 1, 0, 0, 1, 1]),
-            tb.astensor([0, 1, 0, 1, 0, 1, 0, 1]),
-            tb.astensor([0, 0, 0, 0, 1, 1, 1, 1]),
+    if tb.name == "pytorch":
```

Comment:
Is there a more elegant way to do this?

Reply:
probably just pull it out into another test and do `@pytest.mark.only_pytorch` on it?